### ISSUE TITLE
W11 22621.1555 windows 10 ribbon not working

### ISSUE THREAD

**Issue by falcon040** [valinet/ExplorerPatcher#1337]
![1](https://user-images.githubusercontent.com/15073305/232329962-fc96edba-dec5-474d-a334-3cbbc1bf03e4.png)

The windows 10 ribbon or windows7 behaviour is no longer working.

W11 22621.1555

**Comment by topia** [comment]
I don't know the difference, but I can confirm that it works.
How about trying to restart Explorer after changing the setting?
What do you see when you enable "Console" and "Restart File Explorer"?
![image](https://user-images.githubusercontent.com/13070/232330436-4b3d276a-3501-4284-bb4d-c50986a7647e.png)

**Comment by falcon040** [comment]
funchook create 1
Running on Windows 11, OS Build 10.0.22621.1555.
Loaded symbols
Setup ntdll functions done
Setup user32 functions done
Patched taskbar transparency in newer OS builds
Setup explorer functions done
Setup uxtheme functions done
Setup twinui.pcshell functions done
Setup combase functions done
Setup twinui functions done
Setup stobject functions done
Setup bthprops functions done
Setup pnidui functions done
Setup shell32 functions done
Setup explorerframe functions done
Setup windows.storage functions done
[IME] Context menu patch status: 1
Setup inputswitch functions done
Setup windowsudk.shellcommon functions done
Setup peopleband functions done
Installed hooks.
Play startup sound thread...
Signal shell ready...
Started "Signal shell ready" thread.
Open Start on monitor thread
Started "Open Start on current monitor" thread.
EP Service Window thread
Initialized taskbar centering module.
Progman: 197322
Progman hook: 91424281
[Positioning] Added settings for monitor 0000000000010001 : 1
0 0
>>> Signal shell ready.
Setup sndvolsso functions done
Ended "Signal shell ready" thread.
Started "Play startup sound" thread.
Ended "Play startup sound" thread.
[Updates] Starting daemon.
[Updates] Configured update policy on this system: "Check for updates but let me choose whether to download and install them".
[Updates] Path to module: C:\Windows\dxgi.dll
[Updates] Checking against hash "22622.1413.54.5.38505974ea340d7f"
[Updates] Update URL: https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_setup.exe
[Updates] Hash of remote file is "22622.1413.54.5.38505974ea340d7f" (valid).
[Updates] Local version obtained from hash is 22622.1413.54.5.
[Updates] No updates are available.

**Comment by falcon040** [comment]
Its a new installed W11, and yes pressed "Restart Explorer"...

**Comment by topia** [comment]
I compared the log with mine, but sadly, I can't find anything special difference...

**Comment by falcon040** [comment]
I freshly installed w11 22621.1555 in a virtual machine (inlcudinf all updates) and only install explorer patcher, not working with w10 ribbon.

**Comment by EldritchEevee** [comment]
**no longer compatible.**

**Comment by falcon040** [comment]
Installing StartAllBack solved the Problem.

**Comment by falcon040** [comment]
![old explorer look - häckchen weg machen](https://user-images.githubusercontent.com/15073305/235525863-4fa98814-3590-4545-8987-7f9765be03c4.png)

Found the Bug in Windows. If Explorer Patcher is not working on your Sysrem and can not remove the W11 Ribbon, deselect the "Launch folder windows in a seprarate process" and everthing works now.

**Comment by ahikamro** [comment]
I can confirm I had the same issue and the same workaround works,

After the 22H2 update, the option to restore the classic ribbon stopped working for me.

By chance, I discovered yesterday that, at least in my case, it is because I was using the
"launch folder windows in a separate process" option.

Once I uncheck it, EP shows the classic ribbon again.

I reproduced this on a clean Win11 install ( installed latest Win11 as downloaded from Microsoft and then immediately ExplorerPatcher ) and on another install that has been up and running for several months.

Both installs are Win11 Pro.
